docker-compose.yml

```diff
@@ -109,4 +109,82 @@ services:
       - /data/db
     restart: unless-stopped
     networks:
-      - hie
\ No newline at end of file
+      - hie
+
+####
+# OpenELIS
+####
+  certs:
+        container_name: oe-certs 
+        image: ghcr.io/i-tech-uw/certgen:main
+        restart: always
+        environment:
+            - KEYSTORE_PW="kspass"
+            - TRUSTSTORE_PW="tspass"
+        volumes:  
+            - cert-volume:/etc/openelis-global
+          
+  database:
+      container_name: openelisglobal-database 
+      image: postgres:9.5
+      ports:
+          - "15432:5432"
+      restart: always
+      environment:
+          -  POSTGRES_USER=postgres
+          -  POSTGRES_PASSWORD=admin
+          -  POSTGRES_DB=clinlims
+          -  POSTGRES_INITDB_ARGS="--auth-host=md5"
+      volumes:
+            # preserves the database between containers
+            - db-data:/var/lib/postgresql/data                
+            # files here will run on install
+            - openelis-db-data:/docker-entrypoint-initdb.d
```

Comment:
you can get the sql dump files from  https://github.com/I-TECH-UW/OpenELIS-Global-2/tree/develop/dev/database/dbInit